packages/fluentui/react-northstar/src/components/Animation/Animation.tsx

```diff
@@ -231,7 +231,7 @@ const Animation: React.FC<AnimationProps> & {
   const { animationDuration, animationDelay } = animationStyles.root;
   const timeoutResult = timeout || calculateAnimationTimeout(animationDuration, animationDelay) || 0;
 
-  const unhandledProps = getUnhandledProps(Animation.handledProps, props);
+  const unhandledProps = useUnhandledProps(Animation.handledProps, props);
```

Comment:
Should we use the generic typings?

Reply:
Nope, they will be inferred as it was previously for `getUnhandledProps`